lib/decors/method_added.rb

```diff
@@ -5,6 +5,10 @@ module MethodAdded
         module Handler
             private
 
+            METHOD_CALLED_TOO_EARLY_HANDLER = ->(*) {
```

Comment:
If you call `decorated_method.bind(...).call` inside `initialize` in the decorator class, it should raise this error.